src/components/VenuesContainer/VenuesContainer.js

```diff
@@ -1,11 +1,49 @@
 import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import VenueSearchBar from '../VenueSearchBar/VenueSearchBar';
 
-function VenuesContainer() {
+export function VenuesContainer({ venues, isLoading, error }) {
+  const generatedVenues = () => venues
+    .map(venue => (
+      <article className="venue-card" key={venue.id}>
+        <h3>
+          {venue.name}
+        </h3>
+        <p>{venue.notes}</p>
+      </article> 
+    ));
+  
```

Comment:
Should we create a component for the venue cards?

Reply:
We should! This should be a temporary element until for find out more details on how we get back the venues